core/templates/components/forms/custom-forms-directives/custom-form-components.module.ts

```diff
@@ -27,12 +27,16 @@ import { MaterialModule } from 'modules/material.module';
 import { SharedPipesModule } from 'filters/shared-pipes.module';
 import { ImageUploaderComponent } from './image-uploader.component';
 import { TranslateModule } from '@ngx-translate/core';
+import { HybridRouterModuleProvider } from '../../../hybrid-router-module-provider';
 
 @NgModule({
   imports: [
     CommonModule,
     MaterialModule,
     DynamicContentModule,
+    // TODO(#13443): Remove hybrid router module provider once all pages are
+    // migrated to angular router.
+    HybridRouterModuleProvider.provide(),
```

Comment:
Hybrid Router module provider provides RouterModule in shared modules when they are used in pages that are migrated to the angular router and provides MockRouterModule for pages that are not yet migrated.